src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -41,17 +37,17 @@ public void setUp() {
     @Test
     void createStation() {
         // when
-        ExtractableResponse<Response> response = RequestHelper
-                .postRequest(STATION_PATH, new HashMap<>(), Collections.singletonMap("name", "강남역"));
+        String stationName = "강남역";
+        ExtractableResponse<Response> response = StationRequest.createStation(stationName);
```

Comment:
```suggestion
        ExtractableResponse<Response> response = StationRequest.지하철역_생성됨(stationName);
```
테스트 코드는 가독성이 중요하다고 생각되는데요!
그럴경우 저는 한글 메서드명을 사용하는편입니다!

Reply:
넵. 저도 테스트 코드는 해당 테스트만 봐도 어떠한 테스트를 진행하는지 알 수 있어야 한다고 생각합니다.

하지만 회사 업무에서도, 개인 프로젝트, 토이 프로젝트 등에서 진행할 때 한글로 메서드를 사용해 본적이 없습니다. (다른 팀 또는 사용하는 것은 본적이 있습니다.)
그래서 오히려 영어와 한글이 코드에 섞여있으면 개인적으로 더 알아보기가 힘들더라고요.

테스트 명은 한글로 하는 것은 좋다고 생각합니다. 이는 테스트만 보고 어떠한 테스트를 하는지 알 수 있어야 한다에 기초하여 해외 또는 오픈 소스가 아닌 이상 한글을 통해 어떠한 테스트를 진행한다를 명확하게 알 수 있으니까요.

그래서 결론은, 팀의 컨벤션에 의해 해당부분은 충분히 정해질 수 있다고 생각합니다.
만약 테스트 코드의 함수명은 한글로 의도를 명확하게 해야 하는 컨벤션이 존재한다면, 적용하겠습니다.

혹시... 공격적으로 느껴지셨다면... 그런 의도가 아니고 정말 제 개인적인 의견입니다 😂

Reply:
> 혹시... 공격적으로 느껴지셨다면... 그런 의도가 아니고 정말 제 개인적인 의견입니다 

서로의 의견을 공감하고 나누는것이 진정한 코드리뷰가 아닐까요 : )
관점을 공유해주셔서 감사합니다 ^^!